cfgov/jinja2/v1/_includes/organisms/mega-menu.html

```diff
@@ -47,18 +47,27 @@
 
    ========================================================================== #}
 
-{% macro _nav_list( nav_depth, nav_groups, language='en' ) %}
+{% macro _nav_list( nav_depth, nav_groups, nav_title='', language='en' ) %}
     {% for group in nav_groups %}
     <div class="{{ _classes( nav_depth, '-list' ) }}">
+
         {% if group.title %}
+        {% set aria_group_title = group.title ~ ' continued' if group.title_hidden else group.title %}
         <h4 class="h5
                   {{ base_class ~ '_group-heading' }}
                   {{ base_class ~ '_group-heading' ~ '__hidden' if group.title_hidden else ''}}"
-            {% if group.title_hidden %} aria-hidden="true" {% endif %}>
+            id="{{ aria_group_title | slugify ~ '-menu' }}"
+            {% if group.title_hidden %} aria-hidden="true" {% endif %}
+            aria-label="{{ aria_group_title }}">
             {{ group.title }}
         </h4>
         {% endif %}
-        <ul>
+        <ul
+            {% if aria_group_title %}
+            aria-labelledby="{{ aria_group_title | slugify ~ '-menu' }}"
```

Comment:
We end up this:

<img width="423" alt="Screen Shot 2020-03-13 at 12 45 10 PM" src="https://user-images.githubusercontent.com/704760/76641774-80124680-6528-11ea-88b8-604461d48d1d.png">

Is that correct? We have a visible item that is labeled by an aria-hidden item. Will screenreaders handle that?

Reply:
It seems like `aria-labelledby` maybe shouldn't ever point to the `-continued-` instance, but should point to the one that appears in the first column.

Reply:
This is admittedly a little weird. Originally I was printing "Populations Served continued" to the text node and just using `aria-labelledby` for every link group column with no `aria-label`. Buuuuuut then I realized we're using the invisible `h4`s to maintain proper padding above the heading-less columns and by appending "continued" causes a line wrap on smaller viewports. Here's a screenshot with the invisible header revealed in red:

<img width="1054" alt="Screen Shot 2020-03-13 at 1 34 42 PM" src="https://user-images.githubusercontent.com/1060248/76645850-2e6dba00-6530-11ea-9100-b82ec8afeda1.png">

Notice that "Servicemembers & Veterans" is pushed lower than "Adult Financial Education".

So I left the text node as "Populations Served" and added an `aria-label` with the proper text for screenreaders. It looks dumb but the `aria-labelledby` on the `ul` overrides the `aria-hidden` property.

Buuuuut on second thought we can remove the `aria-hidden` logic because we now never want the heading hidden to screenreaders. Problem solved. 👍